airbyte-cdk/bulk/toolkits/load-iceberg-parquet/src/main/kotlin/io/airbyte/cdk/load/command/iceberg/parquet/NessieServerSpecifications.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.command.iceberg.parquet
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonPropertyDescription
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaInject
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
```

Comment:
Are these the correct imports?  If so, we should probably add whatever dependency that they come from to the build.gradle to avoid just relying on transitive dependencies.

Reply:
Thats what the S3 connector was using, I am doing exactly the same thing as what it did.